packages/zend-file-transfer/library/Zend/File/Transfer.php

```diff
@@ -64,18 +63,15 @@ public function __construct($adapter = 'Http', $direction = false, $options = ar
      */
     public function setAdapter($adapter, $direction = false, $options = array())
     {
-        if (Zend_Loader::isReadable('Zend/File/Transfer/Adapter/' . ucfirst($adapter). '.php')) {
-            $adapter = 'Zend_File_Transfer_Adapter_' . ucfirst($adapter);
-        }
-
-        if (!class_exists($adapter)) {
-            Zend_Loader::loadClass($adapter);
-        }
+	    if (class_exists('Zend_File_Transfer_Adapter_' . ucfirst($adapter))) {
```

Comment:
could you adjust the indentation here please?

Reply:
powers of web edit. looks like have to check this out locally. thanks for the reivew.